src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,135 @@
+package com.wolox.training.models;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column
+    private String genre;
+
+    @Column(nullable = false)
+    @NotNull
+    private String author;
+
+    @Column(nullable = false)
+    @NotNull
+    private String image;
+
+    @Column(nullable = false)
+    @NotNull
+    private String title;
+
+    @Column(nullable = false)
+    @NotNull
+    private String subTitle;
```

Comment:
If you name a field with camelCase, you have to use the mapping tag `name = snake_case` as you had before.

Reply:
Solved, name added